endpoints/service.go

```diff
@@ -24,6 +25,8 @@ var (
 	typeOfRequest = reflect.TypeOf((*http.Request)(nil)).Elem()
 	// Precompute the reflect type for Context.
 	typeOfContext = reflect.TypeOf((*Context)(nil)).Elem()
+	// Precompute the reflect type for *VoidMessage.
+	typeOfVoidMessage = reflect.TypeOf(new(VoidMessage))
```

Comment:
Is it possible to do something like this?
typeOfVoidMessage = reflect.TypeOf((*VoidMessage)(nil))